### PR TITLE
Fix for issue #4 (--skipFeatureDeployment not included in spo app deploy command for Windows)

### PR DESCRIPTION
**Pull request by anoopt**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| New sample?      | no
| Related issues?  | #4 

#### What's in this Pull Request?

Added a new function called `cleanString` that removes any spaces and line breaks from a given string. Below is screenshot of how the command will execute now.

image

Sharing is caring.

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -1026,13 +1026,17 @@ function executeO365CLICommand(command) {
...
+            return cleanOutput ? cleanString(o365CLICommandOutput) : o365CLICommandOutput;
         }
         catch (err) {
             throw new Error(err);
         }
     });
 }
+// remove any line breaks and spaces in a string
+function cleanString(input) {
+    return input.replace(/[\r\n\s]+/gm, "");
```
Comment (waldekmastykarz) [inline dist/index.js]: Shall we adjust it to remove only leading/trailing spaces/new lines? If there is a space or a new line inside the string, there is probably a reason for it and removing them could make the output unusable.

**Review by waldekmastykarz — CHANGES_REQUESTED** [review]
Looks good! How about we slightly adjust the clean up logic?

File: dist/index.js (hunk 1 above)
Comment (anoopt) [reply dist/index.js]: Sure @waldekmastykarz I'll push the updated code.

Comment (anoopt) [reply dist/index.js]: @waldekmastykarz - The code is now updated with the `trim()` method which satisfies the requirement of removing leading/trailing spaces/new lines from the output.

**Review by VelinGeorgiev — APPROVED** [review]
Hi @anoopt ,

Thank you for your efforts! I tested it and it works with all linux, mac, and windows while I was able to reproduce the issue without the changes you've made. 

FYI:
I just squashed the changes into one commit and changed the message in the release notes for v1.0.1
I will update the master branch form here: [https://github.com/VelinGeorgiev/action-cli-deploy/tree/fix-emptyline-win](https://github.com/VelinGeorgiev/action-cli-deploy/tree/fix-emptyline-win)

**Comment by VelinGeorgiev** [comment]
The change is published to the marketplace. @anoopt great work as always! Thank you for your time (I know time is precious)! 😃  🚀

**Pull request closed without merging**